metpy/calc/thermo.py

```diff
@@ -395,7 +395,7 @@ def cape(pressure, temperature, dewpt, temperature_parcel, virtual_temp=False):
     x = x[p_mask]
     y = y[p_mask]
 
-    return Rd * np.trapz(y, np.log(x))
+    return (Rd * (np.trapz(y, np.log(x)) * units.degK)).to(units('J/kg'))
```

Comment:
It's really a bummer, but I didn't want to wrap or anything until we figured out what was going on with units. Having the temperature forced to Kelvin at the beginning of the calc ensures this won't break at least.

Also "degrees Kelvin"? Ick! I don't know what we'd call it though. See Wikipedia text:

> "Degrees Kelvin" (K) is a former name for the SI unit of temperature on the thermodynamic (absolute) temperature scale. Since 1967 it has been known simply as the kelvin, with symbol K (without a degree symbol).[1] Degree absolute (°A) is obsolete terminology, often referring specifically to the kelvin but sometimes the degree Rankine as well.

>